### ISSUE TITLE
docs and notebook for sending data to flask needs "combined"

### ISSUE THREAD

**Issue by ageller** [ageller/Firefly#164]
In order to send data to Firefly via flask you need the "combined" (or other flask-enabled) entry point.  The default no longer runs through flask.

**Comment by Zhiyuan-21** [comment]
Thank you for bringing this to my attention! I wonder if this might be the reason behind my difficulties in sending data while accessing remote Firefly servers via port forwarding. I can start the server on the cluster, but cannot loading the data after sending data via flask. Perhaps the issue stems from the need to include “combined” at the end of the URL?

**Comment by ageller** [comment]
Yes, you'll need to use the "combined" endpoint, e.g., see [here](https://github.com/ageller/Firefly/blob/main/src/firefly/ntbks/minimal_example.ipynb).  Also note that there are some updates in the GitHub version that haven't yet been released on pip.  If you still can't send data to the combined endpoint in the version you have installed, you may want to try to current version on GitHub.
